specification/maps/data-plane/Route/preview/1.0/route.json

```diff
@@ -0,0 +1,2563 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Maps Route Service",
+    "version": "1.0",
```

Comment:
I think the version should be a date in YYYY-MM-DD format, suffixed with "-preview" since this is a preview version.

This comment may apply to other files as well.

Reply:
@mikekistler: isn't YYYY-MM-DD versioning for resource-manager? 

Data-plane should be able to version major.minor like other do: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/ComputerVision/stable/v3.1/ComputerVision.json

Reply:
@mikekistler maybe a more relevant example from CV's preview: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview.2/Ocr.json

Reply:
The guidance currently in review and will be likely adopted within the next few weeks is:

> DO use an 'api-version' query parameter with a date value text PUT https://service.azure.com/users/Jeff?api-version=2021-06-04

ref: https://github.com/microsoft/api-guidelines/blob/azureRestUpdates/azure/Guidelines.md#api-versioning